modules/cms/lang/es/lang.php

```diff
@@ -10,15 +10,15 @@
         'invalid_file_extension' => 'Extensión de archivo inválida: :invalid. Las extensiones permitidas son: :allowed.',
         'error_deleting' => 'Error borrando el archivo template ":name". Por favor, revisa los permisos de escritura.',
         'delete_success' => 'Los templates fueron borrados satisfactoriamente: :count.',
-	'file_name_required' => 'Falta el nombre del campo del archivo.',
-	'safe_mode_enabled' => 'Modo seguro esta actualmente activado.'
+    'file_name_required' => 'Falta el nombre del campo del archivo.',
```

Comment:
Why was this merged @daftspunk? This isn't even consistent reformatting, it looks automated to me.